.azure/pipelines/jobs/default-build.yml

```diff
@@ -238,7 +238,7 @@ jobs:
         continueOnError: true
         condition: always()
 
-    - ${{ if and(startsWith(parameters.jobDisplayName, 'Build:'), ne(variables['skipComponentGovernanceDetection'], 'true'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - ${{ if and(startsWith(parameters.jobDisplayName, 'Build:'), ne(parameters.skipComponentGovernanceDetection, 'true'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
@dougbu my question was do we keep the variables['..'] in?

Reply:
Sorry for my late response. I thought I answered this question offline.

Please keep the variable too. That's useful if we decide to turn governance off for an entire pipeline. Could you open another PR to restore that part of the `and(...)`❔

Reply:
It would need to be an `or` no?

Reply:
No the expansion here only happens if _all_ the comma-separated bits evaluate to `true`. Adding another `ne(...)` doesn't change that.

Reply:
Put another way, component governance happens when it's a job named `Build: .*`, the `skipComponentGovernanceDetection` parameter is _not_ `true`, (soon) the `skipComponentGovernanceDetection` variable is not `true`, and this build isn't running to validate a PR.